Smart-Contract-Verifier-Server/src/main/resources/application.properties

```diff
@@ -1,6 +1,6 @@
-spring.datasource.url=jdbc:postgresql://db-postgresql-nyc3-98163-do-user-8880908-0.b.db.ondigitalocean.com:25060/verifierserver?sslmode=require
-spring.datasource.username=${dbuser}
-spring.datasource.password=${dbpassword}
+spring.datasource.url=jdbc:postgresql://localhost:5432/verifierserver
+spring.datasource.username=postgres
+spring.datasource.password=smartcontractserverdev
```

Comment:
No

Reply:
I pushed a new application.properties. This was the original one that was in this branch.